libraries/provision/ansible/playbooks/install-nginx.yml

```diff
@@ -5,6 +5,10 @@
   vars:
     upstream_sync_gatways:
     upstream_sync_gatways_admin:
+    keepalive_timeout: "keepalive_timeout 600s;"
+    proxy_send_timeout: "proxy_send_timeout 600s;"
+    proxy_read_timeout: "proxy_read_timeout 600s;"
```

Comment:
this timeout won't effect SGW load balancer tests, right? If possible, can you run one single topology job with this branch?

Reply:
this is the original default value. if not requiring a customized timeout setting, default will be used.
could you please point me the SGW load balancer tests? upgrade job? I would like to run and verify before merge the code.

Reply:
load balancer job results link added